tests/tests.py

```
@@ -17,7 +20,51 @@ def get_executor(self) -> str:
+    def get_executor(self) -> str:
+        return "slurm"
+
+    def get_executor_settings(self) -> Optional[ExecutorSettingsBase]:
+        return ExecutorSettings(
+            efficiency_report=True,
+            init_seconds_before_status_checks=5,
+            efficiency_report_path=Path("/tmp/efficiency_report_test"),
+            # seconds_between_status_checks=5,
+        )
+
+    def test_simple_workflow(self, tmp_path):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)